book/ui-components/ios/ui-component-ios.md

```diff
@@ -0,0 +1,70 @@
+# UI Component
+
+## How to create new component
+
+1. In `ZLDefines.swift` add the new type of the component under `ZLComponentTypes` enum.
```

Comment:
Where is this file , what repository , who can change it

Reply:
Done.
I added the relevant information to the top of the section.